src/cdk/utils/create-lambda-integration.ts

```diff
@@ -50,6 +56,23 @@ export function createLambdaIntegration(
     }
   );
 
+  const secretsManagerPolicy = new aws_iam.PolicyStatement({
+    actions: ['secretsmanager:GetSecretValue'],
+    resources: [
+      'arn:aws:secretsmanager:' +
+        stack.region +
+        ':' +
+        stack.account +
+        ':secret:*',
+    ],
+  });
+
+  const secretsPolicy = new aws_iam.Policy(stack, 'secrets-getvalue-policy', {
```

Comment:
Please rename variable to `secretsManagerPolicy` and adapt to the project style of naming IDs e.g. 'SecretsManagerPolicy'